designs/virtualization/4.3/vm-details/dedicated-resources/dedicated-resources.md

```diff
@@ -0,0 +1,27 @@
+---
+parent: Virtualization
+version: 4.4
+CNV-version: 2.3
+---
+
+# VM details - dedicated-resources
+
+### note
+The original design was CPU Pinning. It changed to workload Isolation, now it is Dedicated Resources.
+
+## In-page placement
+![VM details - Overview tab](img/in-page-0-1.jpg)
+Within the VM details page > Details tab > Overview section we added the “Dedicated resource” field.
+The default state is “No dedicated resources applied”.
+
+![VM details - Overview tab](img/in-page-0-2.jpg)
+When activated the field will display "Dedicated resources applied"
```

Comment:
Is it possible to list the resources that apply?

Reply:
Interesting. This is a bit tricky since we already specify these resources at the Flavor, one field above.

For non-VM pods, this should be true since the "Dedicated resources" is achieved by matching the 'resources request' values (CPU and Memory) to the 'resources limit', thus not allowing any other workload to use these resources. So, setting the CPU and Memory request and limit should appear on the same modal, which should be called 'Resources management', or something like that.

I wanted to align the VM design to that kind of thinking, unifying 'Flavor' with 'Dedicated resources'. So that unified field can be called 'Resource management' as well with subfields 'Flavor' and 'Dedicated resources'

I eventually chose to keep them split for:
- higher discoverability
- We already have Flavor field placement and that will soon be changed with the upcoming 'Scheduling' section, though I really can't tell when.

But we can try the unified 'Flavor' 'Dedicated resources' design again. Adding an image. 
![Old-VM-dedicated-0-0 Copy](https://user-images.githubusercontent.com/48942365/74655132-88c76500-5194-11ea-8530-58e4c398eba8.jpg)
WDYT?

Reply:
Keeping them split up at least for VMs makes sense to me. Would a label of "[Dedicated CPU Placement](https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/dedicated-cpu.html)" to match the API make sense? I know this thing has been renamed a lot 😄.

I also wonder if "[Quality of Service Class](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/)" should be mentioned somewhere in the future modal to make this feature easier to google for. I don't know enough to tell if that word applies to VM-based pods too though 🤷‍♂️.